test/tools/isoltest.cpp

```diff
@@ -204,13 +216,15 @@ TestStats TestTool::processPath(
 	fs::path const& _basepath,
 	fs::path const& _path,
 	string const& _ipcPath,
-	bool const _formatted
+	bool const _formatted,
+	langutil::EVMVersion const _evmVersion
```

Comment:
No `const`?

Reply:
You see the old one's that are in there above :-)? It wasn't me who added those, but I was too lazy to remove them all, so I kept the "un"-"style" :-)...